data/transport missions.txt

```diff
@@ -1951,3 +1951,369 @@ ship "Science Drone"
+			`	Diana, instead of protesting further, puts her head down. "I'll be in your ship, Captain," she says. Diana storms off, making an effort to create as much noise as possible with the stomps of her feet.`
+			`	"Spoiled kid," Alondo says quietly. "Attitudes like that are why I joined the Southern Defense Pact. Anyway Captain, I suggest you hand her over to the Navy immediately. I'll contact the Republic if you need me to."`
+			choice
+				`	"Alright, I'll hand her over."`
+					goto end
+				`	"Sorry, Alondo, but I agree with her cause."`
+			
+			apply
+				set "navy contacted"
+			`	"I understand. Please, just don't drag the Free Worlds into this."`
+			
+			label end
+			`	You say goodbye to Alondo and return to your ship. When you arrive and look around, Diana is notably absent. In your pilot's seat she has left her backpack and a written note. The paper has noticable tear stains in one of the corners.`
+			``
+			`Dear <first> <last>,`
+			`	I don't care about the credits anymore, but I can't go back to my family. I'll be too much of a disgrace, and I can't even imagine looking my parents in the eyes after what I've done. I've left the credits with you. You can do with them as you wish.`
+			`	I've taken this chance to escape from my old life. I want to become my own person in the world, and I want to do it on my own. Please don't come looking for me, because wherever I am I'll be happier than when I was with my parents. I've failed myself today, but I'm still going to strive for the betterment of the less fortunate. Thank you for trying to make my dreams come true, even if you couldn't succeed.`
+			`	Sincerely,`
+			`				Diana W. Howl`
+			``
+			`	You stare down at the backpack, stuffed full with over half a billion credits.`
+			choice
+				`	(Hand the credits over to the Navy.)`
+				`	(Steal the credits.)`
+					goto steal
+			
+			branch navy
+				has "navy contacted"
+			`	You fold the note and slip it into the backpack so that you can show it to the Navy to explain why Diana is not with you. You contact the Navy's crime report hotline and tell them that you have the stolen credits. A few minutes later, they respond telling you that a fleet is waiting on <destination> to retrieve the credits.`
+				accept
+			
+			label navy
+			apply
+				clear "navy contacted"
+			`	You fold the note and slip it into the backpack so that you can show it to the Navy to explain why Diana is not with you. Alondo sends you a message, telling you that the Navy says they are waiting on <destination> for the credits.`
+				accept
+			
+			label steal
+			`	You rip the note apart and begin counting the credits. In total, the backpack contains a whopping 525,894,400 credits. Over half a billion credits richer, you sit down and wonder what you will spend it on.`
+				decline
+	
+	on accept
+		"reputation: Republic" = 10
```

Comment:
maybe make this `>?=` on the remote chance that the player has since improved their reputation with the republic.